opal/tests/test_managers.py

```diff
@@ -205,3 +210,227 @@ def test_serialised_mine_tag_other_user(self):
             self.user, [self.episode]
         )
         self.assertEqual(as_dict[0]['tagging'][0], {'id': self.episode.id})
+
+
+class LookupListQuerysetTestCase(OpalTestCase):
+    def setUp(self, *args, **kwargs):
+        super(LookupListQuerysetTestCase, self).setUp(*args, **kwargs)
+        self.hat = self.create_model_and_synonym("Top Hat", "High Hat")
+
+    def create_synonym(self, instance, synonym):
+        ct = ContentType.objects.get_for_model(test_models.Hat)
```

Comment:
I get why these three methods are here, but it makes it quite hard to read what exactly we're testing in individual test methods below...